src/routes/transactions/mappers/common/human-descriptions.mapper.spec.ts

```diff
@@ -135,18 +142,28 @@ describe('Human descriptions mapper (Unit)', () => {
   });
 
   it('should return raw amount if token info cannot be fetched', async () => {
-    tokenRepository.getToken.mockImplementation(() => {
-      throw Error('No token info');
+    tokenRepository.getToken.mockImplementationOnce(() => {
+      return Promise.reject();
     });
 
-    const humanDescription = await mapper.mapHumanDescription(
+    const humanDescription = await mapper.mapRichDecodedInfo(
       transaction,
       chainId,
     );
 
-    const shortAddress = mapper.shortenAddress(mockAddress);
-
-    expect(humanDescription).toBe(`Send ${mockAmount} to ${shortAddress}`);
+    const expectedResult = [
+      { type: ValueType.Text, value: 'Send' },
+      {
+        type: ValueType.TokenValue,
+        value: mockAmount.toString(),
+        symbol: null,
+        logoUri: null,
+      },
+      { type: ValueType.Text, value: 'to' },
+      { type: ValueType.Address, value: mockAddress },
```

Comment:
nitpick: use the raw values ("string") as the expected outcome. In doing so this test would fail if `ValueType.Text` (or any other) is changed accidentally.

Reply:
Fair point! I actually changed it to use the enum because of that reason but it also makes sense to have it detect accidental changes to the enum.